lib/iris/tests/runner/_runner.py

```diff
@@ -47,6 +47,7 @@ class TestRunner:
             "The number of processors used for running " "the tests.",
         ),
         ("create-missing", "m", "Create missing test result files."),
+        ("coverage", "l", "Enable coverage testing"),
```

Comment:
We're running out of letters! I chose "l" for "line coverage" as we already use c to mean coding tests

Reply:
I've gone even more extreme: I've removed coding-tests as an option from the test runner. I don't really see the point in keeping it when you could just run the file directly
```
pytest test_coding_standards.py
```

I suppose it is there as a convenience, but I doubt it's being used at the moment?